Jenkinsfile

```diff
@@ -156,7 +155,8 @@ pipeline {
                                     text:"O=0\nCXX=${CXX}")
                             sh """
                                 cd performance-tests-cmdstan
-                                cd cmdstan; make -j${env.PARALLEL} build; cd ..
+
+                                cd cmdstan; make -j6 build; cd ..
```

Comment:
@rok-cesnovar I set this down to 6 because I looked the job was running out of memory. Should we set it to just like half the available `env.PARALLEL`?

Reply:
This was resolved by #858